### PR TITLE
Fixed NextCloud 18 support

### PR DESCRIPTION
**Pull request by modo86**
Fixed NextCloud 18 support

### CODE REVIEW

File: nextcloud-app/lib/auth/abstractzimbrausersbackend.php [hunk 1]
```diff
@@ -61,7 +61,7 @@ public function __construct($server = null, $zimbraAuthenticationBackend = null)
         $this->config = $server->getConfig();
         $this->userManager = $server->getUserManager();
         $this->groupManager = $server->getGroupManager();
-
+/*
```
Comment (modo86) [inline nextcloud-app/lib/auth/abstractzimbrausersbackend.php:64]: NextCloud 18 Support

File: nextcloud-app/lib/auth/abstractzimbrausersbackend.php [hunk 2]
```diff
@@ -70,7 +70,7 @@ public function __construct($server = null, $zimbraAuthenticationBackend = null)
                 $server->getJobList() //Nextcloud >= 12.0.1
             );
         }
-
+*/
```
Comment (modo86) [inline nextcloud-app/lib/auth/abstractzimbrausersbackend.php:73]: NextCloud  18 Support

File: README.md [hunk 3]
```diff
@@ -17,8 +13,7 @@ Features:
 - Attach Nextcloud / ownCloud files to email.
 
 Supported Versions:
-- Nextcloud: 9, 10, 11, 12, 13
-- ownCloud: 9, 9.1, 10
+- Nextcloud: 18
```
Comment (modo86) [inline README.md:16]: NextCloud  18 Support

**Review by modo86 — COMMENTED** [review]
NextCloud  18 Support

**Comment by bud-mo** [comment]
Hello @modo86 and tank you for Your submission!
We cannot merge the edit on the `README.md` file because You added the statement about the fork.

I noticed the drop of the support for OwnCloud and for Nextcloud prior to version 18. Did You managed to perform some tests or You just assumed that the edit can break something?

**Comment by bud-mo** [comment]
Hello @modo86 I am closing this PR as I found the root of the issue.
A `logger` is not passed to the `AccountManager` and this lead to the crash. Merging Your modifications will lead to losing some features.

For more information please see this file: https://github.com/nextcloud/server/blob/stable18/lib/private/Accounts/AccountManager.php#L80

In the next days the fix will reach this repo.

**Comment by modo86** [comment]
Ok. Thank you.

**Comment by bud-mo** [comment]
@modo86 If You are interested, the Nextcloud app on Nextcloud app store has been updated with the support to Owncloud 18.
https://apps.nextcloud.com/apps/zimbradrive

**Comment by adriangibanelbtactic** [comment]
> @modo86 If You are interested, the Nextcloud app on Nextcloud app store has been updated with the support to Owncloud 18. https://apps.nextcloud.com/apps/zimbradrive

Where is the source code for such app version?

**Pull request closed without merging**